test/include/RAJA_test-abs.hpp

```diff
@@ -0,0 +1,32 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-21, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef __RAJA_test_abs_HPP__
+#define __RAJA_test_abs_HPP__
+
+#include "RAJA/RAJA.hpp"
+#include "camp/camp.hpp"
+
+#include <math.h>
```

Comment:
Please change this to <cmath> to follow C++ convention.

Reply:
Done